scripts/quote-board.coffee

```diff
@@ -0,0 +1,42 @@
+# Description:
+#  Quote Board Interface
+#
+# Dependencies:
+#  None
+#
+# Commands:
+#  !quote <name> said <quote> - Adds that quote attributed to that person to the DB
+#  !quote_me_as <name> - Adds <name> to be your full name for quotes attributed to you
+#
+# Notes:
+#  None
+#
+# Author:
+#  @murph33
+
+QUOTE_URL = "https://quote-board.herokuapp.com/api/v1/quotes"
+PERSON_URL = "https://quote-board.herokuapp.com/api/v1/people"
+
+module.exports = (robot) ->
+  robot.hear /![q|Q]uote_me_as (.*)/i, (msg) ->
```

Comment:
`!` is not something that is included in the robot listeners. I'd recommend taking a look at @adammathys jenky integration https://github.com/StemboltHQ/hubot/blob/master/scripts/jenky.coffee

Did you try running hubot locally to test this?